Feature Contactos

larochem20 opened the pull request:
creacion de la carptea coctacos y el archivo contactos.js en la carpeta models

larochem20 committed:
primer commit models/contactos

creacion de la carptea coctacos y el archivo contactos.js en la carpeta models

Endercarias committed:
Crear carpeta contactos en controladores

Se creo la carpeta contactos y dentro de misma el archivo contactos.js

LuisJuarez17 committed:
Creación Rutas Contactos

Creación de la carpeta de rutas para contactos y archivo contactos.js

LuisJuarez17 committed:
Creación de controladores contactos

Se crearon los controladores para contactos

Endercarias committed:
Crear las rutas de contactos

Creación de las rutas de contactos

lighnio commented:
¿Por qué la versión de algunos paquetes cambia?

SFMZ21 reviewed (changes requested):
Rutas y controladores correctos, ¿Por qué hay cambios en las dependencias sequelize y jest? 
¿Por qué se agregó swagger?

DJPORTILLO30 reviewed (changes requested):
Editar destino de pull request ya que va hacia la rama master , necesita ir hacia la rama test-branch! @LuisJuarez17 @larochem20 @Endercarias .

HJOC15 reviewed (changes requested):
Por favor ir agregando comentarios dentro del código.

HJOC15 reviewed (changes requested):
Por favor hacer el cambio de getItems y demás constantes a unas más específicas. Ejemplo: getContactos, createContactos, etc.

LuisJuarez17 committed:
Corrección de versiones de dependencias

Se corrigieron los errores en las versiones de las dependencias de sequelize y jest, además se eliminó la dependencia swagger, ya que se había agregado por error.

Endercarias committed:
Correción de versiones de dependencias (package-lock.json)

Seguimiento de los cambios de las dependencias que se cambiaron en el package.json

LuisJuarez17 committed:
Documentación de controladores (Contactos)

Se realizó la documentación del código en el archivo de los controladores para los contactos.

Endercarias committed:
Documentación de routas de contactos 

Corrección de los nombres de las rutas y se agregaron los comentarios al código

LuisJuarez17 committed:
Creación de modelo de contacto.

Se creó el modelo y se modificaron los controladores de contactos.

Co-Authored-By: Endercarias <50668460+Endercarias@users.noreply.github.com>
Co-Authored-By: larochem20 <70065176+larochem20@users.noreply.github.com>

HJOC15 reviewed (approved):
¡Todo excelente!

DJPORTILLO30 reviewed (changes requested):
Revisar linea #3 de router/contactos.js el import hace referencia a un archivo que no existe asi como la ortografia de Contacts.

SFMZ21 reviewed (changes requested):
Revisar rutas, todo está escrito como "contacs", revisar el import ya que el archivo se llama "contactos"

larochem20 committed:
Corrección de error de referencia 

Se corrigió el error donde se hace referencia a un archivo inexistente, además de la corrección de errores ortográficos en las rutas

DJPORTILLO30 reviewed (approved):
Cambios aprobados!

HJOC15 reviewed (approved):
¡Aprobado!

SFMZ21 reviewed (approved):
Todo ok
